packages/shopware-6-client/src/services/customerService.ts

```diff
@@ -385,3 +386,21 @@ export async function setDefaultCustomerPaymentMethod(
   );
   return response.data;
 }
+
+/**
+ * Checking if user is newsletter subscriber
+ *
+ * @throws ClientApiError
+ * @public
+ */
+export async function isNewsletterSubscriber(
+  contextInstance: ShopwareApiInstance = defaultInstance
+): Promise<{
+  status: string;
```

Comment:
can we expect here some specific values to add it as an enum or union of strings?

Reply:
For now, this endpoint is not described in the documentation, which is why I added string, not enum or union. I think that we can change this type when documentation will be updated and we will be sure about all options